pkg/security/secl/model/marshallers_linux.go

```diff
@@ -120,7 +120,7 @@ func (e *Credentials) MarshalBinary(data []byte) (int, error) {
 }
 
 // MarshalPidCache marshals a binary representation of itself
-func (e *Process) MarshalPidCache(data []byte) (int, error) {
+func (e *Process) MarshalPidCache(data []byte, bootTime time.Time) (int, error) {
```

Comment:
Instead of passing the boot time, we could just give an offset equal to `bootTime.Add(uptime)` ?

Reply:
I'm not sure about the value of passing an offset that will be a negative duration.

Reply:
Nice !